frontend/apps/reader/modules/readerpaging.lua

```diff
@@ -73,6 +99,14 @@ function ReaderPaging:registerKeyEvents()
             args = -1,
         }
     end
+    if Device:hasFiveWay() and not Device:hasKeyboard() then
+        -- targets exclusively kindle 4
+        self.key_events.PrevDocument = {
+            { "ScreenKB", "Back" },
+            event = "PrevDocument",
+            args = 0,
```

Comment:
I believe I shouldn't need this anymore, correct?

Reply:
The args bit? Yes, the handler doesn't take any, so you can elide that. (Same in ReaderRolling).